.pipelines/pipeline.yaml

```diff
@@ -442,6 +442,18 @@ stages:
       vmSize: Standard_B2ms
       dependsOn: "test"
 
+  # Cilium Dualstack Overlay E2E tests
```

Comment:
nit: Can you move this to be next to the other cilium templates. It makes viewing it on ADO much easier. Similarly, if we want to keep dualstack together instead, placing the template here also works.